R/raw_cf.R

```diff
@@ -557,7 +557,7 @@ get_plotdata_raw_cf <- function(cf,
 #' @param reim_same Boolean, determines whether the real and imaginary parts, if both
 #'                  are to be plotted, will be plotted in the same plot.
 #' @param ... Further parameters passed to \link{plotwitherror}.
-#' @return Invisibly returns the plotdata, see \link{get_plotdata.raw_cf}.
+#' @return Invisibly returns the plotdata, see \link{get_plotdata_raw_cf}.
```

Comment:
'.' here, but not in `plot.raw_cf` and others?

Reply:
The choice between `.` and `_` is based on whether a generic function exists or not not. Since `get_plotdata_raw_cf` is a utility function (which is probably not even exported), it makes sense for it to not use the dot notation. On the other hand, `plot.raw_cf` can be dispatched by the `plot` generic function. Is that what you meant?

Reply:
Things are not quite consistent, however, as exemplified by `uwerr`...

Reply:
And there are some generic functions missing, for which it might actually be interesting to provide prototypes.

Reply:
okay, I got it